bin/jmeter.properties

```diff
@@ -917,6 +917,8 @@ summariser.name=summary
 # BackendListener - configuration
 #---------------------------------------------------------------------------
 #
+# Backend metrics window mode (fixed=fixed-size window, timed=time boxed) 
+#backend_metrics_window_mode=fixed
```

Comment:
Documentation is missing for this new feature. It should be mentioned in the properties section and the users manual.